repl/repl.go

```diff
@@ -0,0 +1,312 @@
+package repl
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/d5/tengo/compiler"
+	"github.com/d5/tengo/compiler/ast"
+	"github.com/d5/tengo/compiler/parser"
+	"github.com/d5/tengo/compiler/source"
+	"github.com/d5/tengo/objects"
+	"github.com/d5/tengo/runtime"
+)
+
+const (
+	sourceFileExt = ".tengo"
+	replPrompt    = ">> "
+)
+
+
+//Options represent REPL options
+type Options struct {
+	//Compile output file
+	CompileOutput  string
+
+	//Show help flag
+	ShowHelp       bool
+
+	//Show version flag
+	ShowVersion    bool
+
+	//Input file
+	InputFile      string
+
+	//Builtin modules
+	BuiltinModules map[string]objects.Object
+}
+
+var (
+	bm             map[string]bool
+	builtinModules map[string]objects.Object
+	version        = "dev"
```

Comment:
`version` should be passed by main package as the build (goreleaser) set the version value of main package (`cmd/tengo` and `cmd/tengomin`).